src/collection.js

```diff
@@ -26,6 +26,7 @@ function Collection (manager, options) {
         position: {top: 0, left: 0},
         catchDistance: 200,
         size: 100,
+        nipplesize : 50,
```

Comment:
I'd use a more explicit name for this.
Like `sizeRatio` so we directly understand this option is related to `size` and we can deduct it's a percentage.

And make it `0-1`.